File: firstcode/firstassignemt/myjavaassignment/Palindrome.java
import java.util.Scanner;

public class Palindrome {
	public static void main(String[] args) {
		System.out.println("Welcome to the Palindrome guy");
	
	Scanner input = new Scanner(System.in);

	int number = 0;
	for(int counter = 1; counter <= 10; counter++) {
		System.out.print("Enter a number to check if it is palindrome:  ");
			number = input.nextInt();

		boolean result = palindrome(number);
		if(result == true) {
			System.out.println("the number is palindrome");	
		} else {
			System.out.println("the number is not palindrome");
		}

}
}	
public static boolean palindrome(int number) {
	int firstDivision;
	int secondDivision;
	int thirdDivision;
	int fourthDivision;
	int firstNumber;
	int secondNumber;
	int thirdNumber;
	int fourthNumber;
	int fifthNumber;
	
	
	firstDivision = (number / 10);

	fifthNumber = (number % 10);

	fourthNumber = (firstDivision % 10); 

	secondDivision = (firstDivision / 10);
	thirdNumber = (secondDivision % 10);

	thirdDivision = (secondDivision / 10);
	secondNumber = (thirdDivision % 10);

	fourthDivision = (thirdDivision / 10);
	firstNumber = (fourthDivision);
	
	return (firstNumber == fifthNumber && secondNumber == fourthNumber ? true : false);
}
}

File: firstcode/firstassignemt/newassignement/palindrome.py
def palindrome(number):
	
	first_division = (number // 10)

	fifth_number = (number % 10)

	fourth_number = (first_division % 10) 

	second_division = (first_division // 10)
	third_number = (second_division % 10) 

	third_division = (second_division // 10)
	second_number = (third_division % 10)

	fouth_division = (third_division // 10)
	first_number = (fouth_division)
	
	if first_number == fifth_number and second_number == fourth_number: 
		
		message = "number is palindrome"
		
		return message
	else:
		message = "number is not palinrome"
		return message

for counter in range(10):
	number = int(input("Enter a number to check if it is palindrome:  "))
	result = palindrome(number)
	print(result)


